### ISSUE TITLE
Create Swagger

### ISSUE THREAD

**Issue by drasko** [drasko/edgex-export#21]
Document Export Client and Distro API specs by creating 2 swagger files, each in the appropriate folder.

**Comment by trcox** [comment]
@drasko FYI we currently have RAML API specs in place for the Java services in the `src/test/resources/` directory (copied to wiki) of each service. There are tools to convert between RAML/Swagger, I would say that we have a soft preference for RAML at this point based on support for tools like Postman for API testing. The Java export-distro RAML spec seems to be missing right now, but is minimal with just the `ping` endpoint as far as I can tell.

**Comment by drasko** [comment]
@trcox yep, I just saw it... I was leaning towards Swagger because it is more mature (has been there longer), and because Linux Foundation's [Open API Initiative](https://www.openapis.org/) has chosen Swagger. It became practically de-facto standard for documenting APIs, while RAML is newer competitor.

Because of maturity, Swagger usually has more tool support than RAML, and I can see [here](https://www.getpostman.com/docs/postman/collections/data_formats) that Postman support Swagger out of the box.

On the other hand Intel and OCF have chosen RAML for describing their models (not same as API): https://openconnectivity.org/developer/specifications

We can ask around what community thinks on ML, but the goal of this issue is to remind us that API must be documented, and format I think is not of big importance as we can easily and automatically switch from one to other.
